test/framework/integration_framework/integration_test_framework.cpp

```diff
@@ -534,11 +535,43 @@ namespace integration_framework {
   }
 
   IntegrationTestFramework &IntegrationTestFramework::sendBatch(
-      const std::shared_ptr<shared_model::interface::TransactionBatch> &batch) {
+      const TransactionBatchSPtr &batch) {
     og_transport_->propagateBatch(batch);
     return *this;
   }
 
+  IntegrationTestFramework &IntegrationTestFramework::sendBatches(
+      const iroha::consensus::Round &round,
+      const std::vector<TransactionBatchSPtr> &batches) {
+    auto on_demand_os_transport =
+        iroha::ordering::transport::OnDemandOsClientGrpcFactory(
+            async_call_,
+            [] { return std::chrono::system_clock::now(); },
+            std::chrono::milliseconds(0)  // the proposal waiting
+                                          // timeout does not apply here
```

Comment:
please change the comment.

unfortunately, the comment brings more confusion than clarification.
Please add the reason, **why** timeout does not apply here.

without the clarification, the comment just states the fact without explanation